Tests/configure_and_test_integration_instances.py

```diff
@@ -124,19 +126,23 @@ def reconnect_client(self):
                                                  verify_ssl=False,
                                                  api_key=self.api_key,
                                                  auth_id=self.xdr_auth_id)
+        custom_user_agent = f"demisto-py/dev (Build:{self.build_number})"
+        self.__client.api_client.user_agent = custom_user_agent
+        logging.debug(f'Setting user agent on client to:{custom_user_agent}')
```

Comment:
Log before the action (also because the tense in the log message isn't a past tense...)

```suggestion
        logging.debug(f'Setting user agent on client to:{custom_user_agent}')
        custom_user_agent = f"demisto-py/dev (Build:{self.build_number})"
        self.__client.api_client.user_agent = custom_user_agent
```